pkg/codegen/hcl2/model/print_test.go

```diff
@@ -0,0 +1,26 @@
+package model
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func TestPrintNoTokens(t *testing.T) {
+	b := &Block{
+		Type: "block", Body: &Body{
+			Items: []BodyItem{
+				&Attribute{
+					Name: "attribute",
+					Value: &LiteralValueExpression{
+						Value: cty.True,
+					},
+				},
+			},
+		},
+	}
+	expected := "block {\n    attribute = true\n}"
```

Comment:
Should this have a trailing newline after the closing `}`?

Reply:
No, this is okay--it's the responsibility of the body printer to print the trailing newline, and this is just printing a block. If we were printing a body, this would get a trailing newline.